sdk/math/wasm_helpers.rs

```diff
@@ -81,3 +90,39 @@ macro_rules! resolve {
         }
     }};
 }
+
+#[wasm_bindgen(js_name = "calculateTokenAmountsFromPosition")]
+pub fn calculate_token_amounts_from_position_liquidity(
```

Comment:
I was wondering whether such functions should be defined in TS or Rust, and we should create function in TS. Inside call `calculateAmountDelta` wasm function.

Why use TS?
1. There is no duplicating math logic here (so no risk of inconsistency)
2. This function doesn't calculate anything, it's just wrapping the interface. In TS it can be done much faster and we can expect the same result.
3. From the sdk client's perspective, we should build easy to use and high-level interface.
E.g. this interface looks much better:
```Typescript
const { x, y } = calculateTokenAmountsFromPosition(pool, position)
```
than this one:
```Typescript
const { x, y } = calculateTokenAmountsFromPosition(
  pool.currentTickIndex,
  pool.sqrtPrice,
  position.liquidity,
  position.upperTickIndex,
  position.lowerTickIndex,
  position.tokensOwedX,
  position.tokensOwedY
)
```